api/src/ServicePoller.h

```diff
@@ -20,6 +20,7 @@ struct ChannelState {
   std::string thumbnail;
   uint64_t viewers;
   bool live;
+  bool nsfw;
```

Comment:
this needs to be added to the stream here https://github.com/MemeLabs/Rustla2/blob/master/api/src/ServicePoller.cpp#L44-L48

streams should have a new property called `service_nsfw` so we can continue to manually mark streams with !modify. in the strims api the `nsfw` property can be true if either the internal or service nsfw values are true.